src/components/call/CallGeneralInfo.tsx

```diff
@@ -90,9 +167,77 @@ const CallGeneralInfo: React.FC<{
         />
         <Field
           name="referenceNumberFormat"
+          validate={validateRefnumber}
           label="Reference number format"
           type="text"
           component={TextField}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton onClick={handleClickOpen}>
+                  <HelpIcon />
+                </IconButton>
+                <Dialog
+                  onClose={handleClose}
+                  aria-labelledby="customized-dialog-title"
+                  open={open}
+                  color="primary"
+                >
+                  <DialogContent dividers>
+                    <Typography gutterBottom color="inherit" variant="body1">
+                      A reference number format determines how reference numbers
+                      are generated. It consists of a <strong>prefix </strong>
+                      and <strong> digits parameter</strong>.<br></br>
+                      <br></br>
+                      The<strong> prefix</strong> can contain alphanumeric
+                      characters and is what all generated reference numbers
+                      will begin with. For example, <code>21a</code>.<br></br>
+                      <br></br>
+                      The <strong> digits parameter</strong> is a numerical
+                      value that determines how many digits a proposal&apos;s
+                      sequence number is, including padding.It is written as
+                      <code>{'{digits:x}'}</code>, where x is a the value of the
+                      number. For example, if parameter is 6 (
+                      <code>{'{digits:6}'}</code>),the first proposal will be
```

Comment:
```suggestion
                      <code>{'{digits:6}'}</code>), the first proposal will be
```
![image](https://user-images.githubusercontent.com/31534888/125298511-1d011180-e320-11eb-9e4c-0871775c4604.png)